tests/run-bank.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# TODO: update existing script.
+# Temporary since a custom oasis-node version is needed.
+
+set -o nounset -o pipefail -o errexit
+trap "exit 1" INT
+
+# Oasis Core release version to test against.
+OASIS_RELEASE=21.0.1
+
+# Get the root directory of the tests dir inside the repository.
+ROOT="$(cd $(dirname $0); pwd -P)"
+
+# What to use for GNU tar.
+TAR=tar
+
+# ANSI escape codes to brighten up the output.
+RED=$'\e[31;1m'
+GRN=$'\e[32;1m'
+CYAN=$'\e[36;1m'
+OFF=$'\e[0m'
+
+# The base directory for all the node and test env cruft.
+TEST_BASE_DIR=$(mktemp -d -t oasis-sdk-e2e-XXXXXXXXXX)
+
+# Kill all dangling processes on exit.
+cleanup() {
+	printf "${OFF}"
+	pkill -P $$ || true
+	wait || true
+}
+trap "cleanup" EXIT
+
+# Find build tools.
+if [[ "$(which go)" == "" ]]; then
+	printf "${RED}### Please install 'go'.${OFF}\n"
+	exit 1
+fi
+if [[ "$(which cargo)" == "" ]]; then
+	printf "${RED}### Please install 'cargo'.${OFF}\n"
+	exit 1
+fi
+
+# Find a downloader tool.
+if [[ "$(which wget)" == "" ]]; then
```

Comment:
```suggestion
if ! has_cmd wget; then
```

Reply:
Yeah this is just a copy of the existing `run-e2e.sh` script hacked to work with the custom oasis-core i need, that i plan to remove :-)
 
You'll probably want to file these as an issue to clean up the`run-e2e.sh`?